src/winreg_helper.py

```diff
@@ -0,0 +1,429 @@
+#!/usr/bin/env python
```

Comment:
what is the source of this? If this is on PYPI it should be get from there. 

I know this is quite thin and useful wrapper, but prefer to not adding unnecessary changes are they need extensive tests.

I would appreciate if you remove this (at least from this PR). We shouldn't change to much things at once (see https://medium.com/@fagnerbrack/one-pull-request-one-concern-e84a27dfe9f1 )

Reply:
OK

Reply:
OK, this PR not changed that.

Reply:
removed